packages/proxy-agent/package.json

```diff
@@ -80,6 +80,7 @@
     "@types/dns-packet": "5.6.4",
     "@types/eslint": "8.44.2",
     "@types/mocha": "10.0.1",
+    "@types/node": "20.11.19",
```

Comment:
Should use the same version for all packages -- whether that be 18.x or 20.x

Reply:
I believe in this PR I used `20.11.19` everywhere -- not because I chose this version of node but because it's what we already had resolving in the code. Happy to change it though!